policyengine_us/parameters/gov/states/nh/tax/income/in_effect.yaml

```diff
@@ -0,0 +1,21 @@
+description: New Hampshire taxes interest and dividend income, if this is true.
```

Comment:
In case they expand the base in the future (or did in the past) we can make this more general (label too)
```suggestion
description: New Hampshire taxes individual income, if this is true.
```